sns/json_schemas/operation.json

```diff
@@ -97,6 +97,9 @@
         "filterPolicy": {
           "$ref": "#/definitions/filterPolicy"
         },
+        "filterPolicyScope": {
```

Comment:
As `filterPolicy` and `filterPolicyScope` will be properties *directly* on the object they don't need to be `$ref`'d.